BasicSample/Assets/ARAnchor/ARAnchor.unity

```diff
@@ -594,12 +594,12 @@ PrefabInstance:
     - target: {fileID: 2478658878998671300, guid: dfa794c1410554c44baf341ace4e9495, type: 3}
       propertyPath: m_Text
       value: "AR Anchors and Anchor Persistence\n\n- Air tap anywhere to create an
-        AR\r Anchor, represented as blue cube.\n- Air tap at an anchor's cube to
-        toggle anchor\r persistence. A green cube represents that the anchor is being
-        persisted in the Anchor\r Store.\n- The \"Clear Anchor Store\" button removes
-        all persisted anchors\r from the Anchor Store. All green cubes would turn
-        blue.\n- The \"Clear all\r anchors in scene\" button deletes all anchors
-        in the scene. All cubes will disappear but persistence is preserved.\n"
+        AR Anchor, represented as blue cube.\n- Air tap at an anchor's cube to
+        toggle anchor persistence. A green cube represents that the anchor is being
+        persisted in the Anchor Store.\n- The \"Clear Anchor Store\" button removes
+        all persisted anchors from the Anchor Store. All green cubes would turn
+        blue.\n- The \"Clear all anchors in scene\" button deletes all anchors
+        in the scene. All cubes will disappear but persistence is preserved."
```

Comment:
@devdeeprc this is what was making the text panel resize weird for this text, btw.
Shouldn't be an issue if there are any more '\r's laying around, but I cleaned this up anyways.